dlt/helpers/airflow_helper.py

```diff
@@ -154,22 +154,26 @@ def add_run(
         Args:
             pipeline (Pipeline): An instance of pipeline used to run the source
             data (Any): Any data supported by `run` method of the pipeline
-            decompose (Literal["none", "serialize"], optional): A source decomposition strategy into Airflow tasks. Defaults to "none".
+            decompose (Literal["none", "serialize", "parallel"], optional):
+                A source decomposition strategy into Airflow tasks. Defaults to "none".
+                serialize - decompose the source into a sequence of Airflow tasks.
+                parallel - decompose the source into a parallel Airflow task group (NOTE:
```

Comment:
right - but this is relevant for people running dags locally. so I'd just warn that in case of SequentialExecutor everything is serialized anyway. There are many many other executors that local - it is not the only alternative